examples/javascript/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "javascript-example",
+  "scripts": {
+    "test": "jest",
+    "fuzz": "JAZZER_FUZZ=1 jest",
+    "coverage": "jest --coverage"
+  },
+  "devDependencies": {
+    "@jazzer.js/jest-runner": "^1.3.0"
```

```suggestion
    "@jazzer.js/core": "^1.3.0",
    "@jazzer.js/jest-runner": "^1.3.0",
    "@types/jest": "^29.4.0",
    "jest": "^29.4.3"
```
To avoid IDE warnings about unresolved jest functions and the Jazzer.js core package not being installed (the example uses the fuzzed data provider from the core package)